### PR TITLE
doc: add usage example

### LINKED ISSUE #21: usage example
I created a usage example similar to the one in the libp2p/js-libp2p-tcp readme for myself. Would it be interesting to add such an example to the readme for this repo? If so, I can create a PR.

### PR DESCRIPTION
**Pull request by backkem**
Adds usage example to match libp2p/js-libp2p-tcp.

closes #21.

### CODE REVIEW

**Comment by backkem** [comment]
I'm not sure what Jenkins is falling over. @vasco-santos maybe you can have a look?

**Comment by vasco-santos** [comment]
Thanks for your PR @backkem 

I will restart Jenkins for you and review your PR

File: README.md [hunk 1]
```diff
@@ -20,26 +20,84 @@
...
+## Usage
+
+```js
+const WebRTCDirect = require('libp2p-webrtc-direct')
+const multiaddr = require('multiaddr')
+const pull = require('pull-stream')
+
+const mh = multiaddr('/ip4/127.0.0.1/tcp/9090/http/p2p-webrtc-direct')
+
+const direct = new WebRTCDirect()
```
Comment (vasco-santos) [inline README.md]: Can you change for this:

```js
const webRTCDirect = new WebRTCDirect()
```

I think direct is kind of an unclear name.

**Review by vasco-santos — CHANGES_REQUESTED** [review]
Thank you for the PR @backkem 

I requested just a small change and then I will be able to merge it.

**Comment by vasco-santos** [comment]
Hello @backkem 

Please rebase your PR with the new commits in master. After that, CI will hopefully be green again 🚀

**Comment by backkem** [comment]
Re-based. Seems like the CI is completely gone now.

**Review by vasco-santos — APPROVED** [review]

**Pull request merged**